cdc/log.cc

```diff
@@ -1302,6 +1303,13 @@ class transformer final {
                     }
                     return make_lw_shared<cql3::untyped_result_set>(*result_set);
         });
+      } catch (exceptions::unavailable_exception& e) {
+        // `query` can throw `unavailable_exception`, which is seen by clients as ~ "NoHostAvailable". 
+        // So, we'll translate it to a `read_failure_exception` with custom message.
+        cdc_log.debug("Preimage: translating a (read) `unavailable_exception` to `request_execution_exception` - ", e);
```

Comment:
I think you need to add `{}` after the dash, otherwise the `e` parameter will never get printed, right?

Reply:
ooops! done.